### DISCUSSION TITLE
Profile can have the same name

### DISCUSSION THREAD

**Discussion by prury** [kodadot/nft-gallery#10497] [Q&A]
as @JustLuuuu pointed out, there is no system for verifying if a profile has the same name of another profile.

is it worth to implement? how hard it is?

**Comment by vikiival** [comment]
well,
I would suggest to have `handle` that would be unique, 
I do not see an issue having two people with same name (aka I can have name Elon Musk on X, but I cannot have handle `elonmusk`)

**Reply by JustLuuuu** [reply]
Yes, having a handle is a nice solution. I agree with the fact that names can be the same and only handle would be unique. But since we have priorities, we have to think if this is something that will take a lot of our time to add/implement.

And also ...

What will the function of the handle be? or is it just a cosmetic thing and to have a sense of uniqueness? Search users still don't work on Koda right now, but maybe in the future we can add that too?

**Reply by vikiival** [reply]
> What will the function of the handle be? 
Universality

`koda.art/u/@JustLuu`
